### PR TITLE
Fixed Webpack configuration

### PR DESCRIPTION
**Pull request by AlexanderEkdahl**
@enginehenryed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>V tag: a great application to use</title>
```
Comment (AlexanderEkdahl) [inline index.html:6]: Should change this

Comment (AlexanderEkdahl) [reply index.html:6]: To klotter

**Review by AlexanderEkdahl — COMMENTED** [review]
Fixes #37

File: index.html (hunk 1 above)
Comment (eldenpark) [reply index.html:6]: will work on that

**Pull request merged**